### DISCUSSION TITLE
iOS: Click on "Enable auto-merge" does not update UI

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#4064] [Mobile] [answered]
Clicking on "Enable auto-merge" does not update the UI, you have to refresh the page/PR manually.

https://user-images.githubusercontent.com/22521688/120171844-bf4ab700-c202-11eb-82da-3721b8c3333d.mov

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report @hfhbd – I was able to reproduce this issue on my side as well, and I've opened up a tracking issue for our team to investigate a fix 😊

**Comment by aconsuegra** [comment]
Hey @hfhbd. We have fixed this issue and the fix will be available in our upcoming app update!
